### PR TITLE
[Dynaconf] Compute Resource Libvirt

### PR DESCRIPTION
**Pull request by akhil-jha**
GitLab MR: 164
```
$ pytest -k test_positive_create_with_name tests/foreman/api/test_computeresource_libvirt.py
==================================================================== test session starts =====================================================================
platform linux -- Python 3.8.7, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: robottelo, configfile: pyproject.toml
plugins: reportportal-5.0.8, forked-1.3.0, xdist-2.2.1, mock-3.5.1, ibutsu-1.14.1, services-2.2.1
collected 67 items / 60 deselected / 7 selected                                                                                                              

tests/foreman/api/test_computeresource_libvirt.py .......                                                                                              [100%]

============================================================= 7 passed, 60 deselected in 34.48s ==============================================================

```

### CODE REVIEW

File: conf/libvirt.yaml.template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+COMPUTE_RESOURCES:
+  # External Libvirt Hostname
+  LIBVIRT_HOSTNAME: libvirt.example.com
```
Comment (omkarkhatavkar) [inline conf/libvirt.yaml.template:3]: if these values are going to get updated based on git lab PR, we should comment those attributes

File: conf/libvirt.yaml.template [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  # Path on the Libvirt host, where the virtual images will be stored.
+  # The default libvirt image dir will be used i.e., "/var/lib/libvirt/images/".
+  LIBVIRT_IMAGE_DIR: /path/to/libvirt/images
```
Comment (omkarkhatavkar) [inline conf/libvirt.yaml.template:6]: we can pass actual libvirt directory path here nothing I sense as sensitive

**Review by omkarkhatavkar — CHANGES_REQUESTED** [review]

**Review by omkarkhatavkar — APPROVED** [review]
ACK

File: conf/libvirt.yaml.template [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  # LIBVIRT_HOSTNAME: libvirt.example.com
+  # Path on the Libvirt host, where the virtual images will be stored.
+  # The default libvirt image dir will be used i.e., "/var/lib/libvirt/images/".
+  LIBVIRT_IMAGE_DIR: /var/lib/libvirt/images
+  # For discovery ISO
+  # Discovery ISO name is required for PXE-less discovery tests. ISO's are
+  # rebuilt with extra kernel options for unattended pxe-less discovery
+  # ISO file names should be like below for downstream_el6 and el7 environments:
+  # foreman-discovery-image-3.0.5-3_unattended_down_el7.iso
+  # foreman-discovery-image-3.0.5-3_unattended_down_el6.iso
```
Comment (JacobCallahan) [inline conf/libvirt.yaml.template:12]: looks like you're missing an empty line at the end of this file

**Review by JacobCallahan — CHANGES_REQUESTED** [review]

**Review by JacobCallahan — APPROVED** [review]

**Pull request merged**